src/antsibull/data/ansible-setup_py.j2

```diff
@@ -173,7 +173,8 @@ setup(
     packages=['ansible_collections'],
     include_package_data=True,
     install_requires=[
-        '{{ ansible_core_package_name }}>={{ ansible_base_version }},<{{ ansible_base_version.major }}.{{ ansible_base_version.minor + 1 }}',{{ collection_deps }}
+        '{{ ansible_core_package_name }} ~= {{ ansible_base_version }}',
+        {{ collection_deps }}
```

Comment:
```suggestion
        '{{ ansible_core_package_name }} ~= {{ ansible_base_version }}',{{ collection_deps }}
```
In the `ansible` package build, `collection_deps == ''`, so having it in its own line looks ugly. Also in case it is non-empty, it will already contain leading spaces and newlines so that the result looks good.

Reply:
Well, I'd argue that it's the source, meaning this template, we need to care about when judging whether something is ugly, not autogenerated files.
Plus, it's probably reasonable to actually pass a list and have a `for`-loop in the template. But it's not worth the trouble right now so I'll accept your suggested change.